SDWebImage/Core/NSButton+WebCache.h

```diff
@@ -86,7 +88,7 @@
  *                       The fourth parameter is the original image url.
  */
 - (void)sd_setImageWithURL:(nullable NSURL *)url
-                 completed:(nullable SDExternalCompletionBlock)completedBlock;
+                 completed:(nullable SDExternalCompletionBlock)completedBlock NS_SWIFT_NAME(sd_setImage(url:completed:));
```

Comment:
`completed` sounds like a boolean to me. Maybe `completion` (very common name in Swift) would be a better choice?

Reply:
@dreampigg How about standard complete block name both OC & Swift in 6.0 version.